Auth0/Auth0Authentication.swift

```diff
@@ -476,6 +476,10 @@ struct Auth0Authentication: Authentication {
                        telemetry: self.telemetry)
     }
 
+    func customTokenExchange(subjectTokenType: String, subjectToken: String, scope: String = "openid profile email", audience: String? = nil) -> Request<Credentials, AuthenticationError> {
```

Comment:
This should be using the [defaultScope](https://github.com/auth0/Auth0.swift/blob/2.13.0/Auth0/Auth0.swift#L33) property instead of hardcoding it.